pyiron_atomistics/calphy/base.py

```diff
@@ -0,0 +1,602 @@
+    
+    An input structure (:attr:`structure`) and interatomic potential (:attr:`potential`) are necessary input options. The additional input options such as the temperature and pressure are specified in the :meth:`.calc_free_energy` method. Depending on the input parameters, a corresponding calculation mode is selected. Further input options can be accessed through :attr:`input.md` and :attr:`input.tolerance`. 
+    
+    An example which calculates the free energy of Cu using an interatomic potential:
+    
+    ```python
+    job.structure = pr.create.structure.ase.bulk('Cu', cubic=True).repeat(5)
+    job.potential = "2001--Mishin-Y--Cu-1--LAMMPS--ipr1"
+    job.calc_free_energy(temperature=1100, pressure=0, reference_phase="solid")
+    job.run()
+    ```
+    
+    In order to calculate the free energy of the liquid phase, the `reference_phase` should be set to `liquid`. 
+    
+    The different modes can be selected as follows:
+    
+    For free energy at a given temperature and pressure:
+
+    ```python
+    job.calc_free_energy(temperature=1100, pressure=0, reference_phase="solid")
+    ```
+    
+    To obtain the free energy between a given temperature range (temperature scaling):
+    
+    ```python
+    job.calc_free_energy(temperature=[1100, 1400], pressure=0, reference_phase="solid")
+    ```
+    
+    For free energy between a given pressure range (pressure scaling)
+    
+    ```python
+    job.calc_free_energy(temperature=1000, pressure=[0, 100000], reference_phase="solid")
+    ```
+    
+    To obtain the free energy difference between two interatomic potentials (alchemy/upsampling)
+    
+    ```python
+    job.potential = ["2001--Mishin-Y--Cu-1--LAMMPS--ipr1", "1986--Foiles-S-M--Cu--LAMMPS--ipr1"]
+    job.calc_free_energy(temperature=1100, pressure=0, reference_phase="solid")
+    job.run()
+    ```
+    
+    The way `pressure` is specified determines how the barostat affects the system. For isotropic pressure control:
+    
+    ```python
+    job.calc_free_energy(temperature=[1100, 1400], pressure=0, reference_phase="solid")
+    ```
+    
+    For anisotropic pressure control:
+
+    ```python
+    job.calc_free_energy(temperature=[1100, 1400], pressure=[0, 0, 0], reference_phase="solid")
+    ```
+    
+    To constrain the lattice:
+
+    ```python
+    job.calc_free_energy(temperature=[1100, 1400], pressure=None, reference_phase="solid")
+    ```
+    
+    In addition the boolean option :attr:`input.npt` can be used to determine the MD ensemble. If True, temperature integration and alchemy/upsampling are carried out in the NPT ensemble. If False, the NVT ensemble is employed.
+    
+    After the calculation is over, the various output options can be accessed through `job.output`.
+    """
+    def __init__(self, project, job_name):
+        super(CalphyBase, self).__init__(project, job_name)
+        self.__name__ = "CalphyJob"
```

Comment:
```suggestion
        self.__name__ = "CalphyJob"
```
No need, done by `GenericJob` automatically.